### PR TITLE
feat: improved folder cohesion

### PR DESCRIPTION
**Pull request by jguddas**

### CODE REVIEW

**Review by ericfennis — APPROVED** [review]
Nice changes, looks perfect to me!

File: icons/folder-symlink.svg
Comment (karsa-mistmere) [inline icons/folder-symlink.svg:1]: Not sure about this one, it's not consistent with `file-symlink` and changes the meaning a bit too much imo.

File: icons/folder-heart.svg
Comment (karsa-mistmere) [inline icons/folder-heart.svg:1]: The heart shape fell victim to overoptimization and is not pixel perfect anymore.

File: icons/folder-output.svg
Comment (karsa-mistmere) [inline icons/folder-output.svg:1]: Also not consistent with `file-output`.

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

**Comment by jguddas** [comment]
@karsa-mistmere do you want me to drop the files that you added comments to?

File: icons/folder-heart.svg [hunk 1]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M11 20H4a2 2 0 0 1-2-2V5c0-1.1.9-2 2-2h3.93a2 2 0 0 1 1.66.9l.82 1.2a2 2 0 0 0 1.66.9H20a2 2 0 0 1 2 2v1.5" />
-  <path d="M21.29 13.7a2.43 2.43 0 0 0-2.65-.52c-.3.12-.57.3-.8.53l-.34.34-.35-.34a2.43 2.43 0 0 0-2.65-.53c-.3.12-.56.3-.79.53-.95.94-1 2.53.2 3.74L17.5 21l3.6-3.55c1.2-1.21 1.14-2.8.19-3.74Z" />
+  <path d="M11 20H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h3.9a2 2 0 0 1 1.69.9l.81 1.2a2 2 0 0 0 1.67.9H20a2 2 0 0 1 2 2v1.5" />
+  <path d="M21.1 17.5a2.5 2.5 0 1 0-3.26-3.79l-.34.39-.3-.4a2.5 2.5 0 1 0-3.29 3.75L17.5 21Z" />
```
Comment (karsa-mistmere) [inline icons/folder-heart.svg:13]: This is the best I could do that didn't affect the shape and size too much:
```suggestion
  <path d="M13.9 17.45c-1.2-1.2-1.14-2.8-.2-3.73a2.43 2.43 0 0 1 3.44 0l.36.34.34-.34a2.43 2.43 0 0 1 3.45-.01v0c.95.95 1 2.53-.2 3.74L17.5 21Z" />
```

**Review by karsa-mistmere — COMMENTED** [review]
> @karsa-mistmere do you want me to drop the files that you added comments to?

Not necessarily, but all of those issues need to be addressed in some way:

* `folder-heart`: the icon could definitely be better optimized, but that needs some tweaking for heart, for which I've added a suggestion
* `folder-symlink`: either file-symlink will have to be updated to match, or `folder-symlink` needs to remain consistent with `file-symlink` (optimisation aside)
* the same is true for `folder-output` and `file-output`: I'm fine with either design but I think it would need to be consistent throughout all file and folder icons, this affects a **_lot_** of icons, namely `file-code/archive/audio/check-2/code-2/cog/digit/heart/input/json-2/key-2/lock-2/plus-2/minus-2/search/type-2/video-2/volume/x-2`.

**Comment by ericfennis** [comment]
@jguddas Can you rebase this PR and take a look at the comments from @karsa-mistmere ?

**Review by karsa-mistmere — APPROVED** [review]

**Pull request merged**